config/applitools.config.js

```diff
@@ -0,0 +1,44 @@
+const { execSync } = require('child_process');
+const merge = require('lodash/merge');
+const { DATA_READY_HOOK } = require('../src/hooks');
+
+const PULL_REQUEST_PARENT_HASH_INDEX = 2;
+const HEAD_HASH_INDEX = 0;
+
+function getHeadHash() {
+  return execSync('git rev-parse --verify HEAD').toString();
+}
+
+function getParentsHashArray() {
+  const headCommitHash = getHeadHash();
+  return execSync(`git rev-list --parents -n 1 ${headCommitHash}`)
+    .toString()
+    .split(' ');
+}
+
+function getPRHeadHash() {
+  const parentsHashArr = getParentsHashArray();
+  const isPullRequest = parentsHashArr.length === 3;
+  const parentHashIndex = isPullRequest
+    ? PULL_REQUEST_PARENT_HASH_INDEX
+    : HEAD_HASH_INDEX;
+  return parentsHashArr[parentHashIndex].trim();
+}
+
+function getBatchId() {
```

Comment:
On the one hand, we do want a common utility that will eliminate the code duplication with this configuration, but I wonder if it's the `storybook-snapper` responsibility.

Also, `process.env.BUILD_VCS_NUMBER` is an internal CI thing and not something a real opensource will use.

WDYT?

Reply:
Yeah, I agree.
So what we're saying, is that this config should just extend the applitools' configuration with the `eyes-storybook` configuration.
So the whole file would just be:
```js
module.exports = ({config}) => merge({
  waitBeforeScreenshots: `[${DATA_READY_HOOK}="true"]`,
}, config);
```

Reply:
But then it's not that different than my initial solution, no?